includes/Modules/Analytics_4.php

```diff
@@ -1088,7 +1078,7 @@ protected function sync_google_tag_settings() {
 	 * @throws Invalid_Param_Exception Thrown if a parameter is invalid.
 	 * @throws Missing_Required_Param_Exception Thrown if a required parameter is missing or empty.
 	 *
-	 * phpcs:ignore Squiz.Commenting.FunctionCommentThrowTag.WrongNumber
+     * phpcs:ignore Squiz.Commenting.FunctionCommentThrowTag.WrongNumber
```

Comment:
I'll fix the indentation for this line:

```suggestion
   * phpcs:ignore Squiz.Commenting.FunctionCommentThrowTag.WrongNumber
```